app/models/finance/schedule.rb

```diff
@@ -4,7 +4,15 @@ class Finance::Schedule < ApplicationRecord
   has_many :milestones, -> { order(milestone_date: :asc) }
   has_many :participant_profiles
 
-  def self.default
+  def self.default_ecf
```

Comment:
Would it not be better to index these by course identifier like it's done in the following "accept" and then just call default (or default_for()) with a course identifier.